### DISCUSSION TITLE
Add an easy way to copy debug information

### DISCUSSION THREAD

**Discussion by maxpatiiuk** [specify/specify7#5585] [Feature Requests]
When someone internal creates a new issue, they need to attach the current database name, current page URL, history of previous pages visited, user name and collection name.
It would save a lot of time if there was a keyboard shortcut or a button to copy that information

Blocked by https://github.com/specify/specify7/issues/1746
Related to https://github.com/specify/specify7/issues/1466

**Comment by grantfitzsimmons** [comment]
Keyboard shortcut for this would be awesome
